packages/perseus/src/validation.types.ts

```diff
@@ -188,7 +187,10 @@ export type PerseusRadioUserInput = {
     noneOfTheAboveSelected?: boolean;
 };
 
-export type PerseusSorterRubric = PerseusSorterWidgetOptions;
+export type PerseusSorterRubric = {
+    // Translatable Text; The correct answer (in the correct order). The user will see the cards in a randomized order.
```

Comment:
This comment is a little confusing to me, is it pulled from somewhere?

Reply:
Yes, this comment is included within the `PerseusSorterWidgetOptions`. I copy the whole block of options and then whittle it down to only what's needed. I could just remove the comment from here since it's still in the options object?